test/reporters/base.test.ts

```diff
@@ -612,28 +616,24 @@ describe('BaseReporter', () => {
         const price = await reporter.priceToReport()
         expect(currentPriceFn).toBeCalledTimes(1)
         expect(price).toBe(currentPriceValue)
-        expect(reporter.circuitBreakerOpen).toBe(false)
```

Comment:
Because I've removed the `_circuitBreakerOpen ` member from `BaseReporter`. There's no need for an explicit flag, as the conditions for the circuit breaker being open depend only on the last reported price and time.

Reply:
I'd leave the flag though, so we can export a metric out of it. WDYT?

Reply:
We don't have a metric for that, and if we did I don't think having this member would be particularly helpful (we could set the metric value every time an oracle target block arrives).